pkg/lvm/lvm_util.go

```diff
@@ -49,29 +49,38 @@ const (
 	VGCreate = "vgcreate"
 	VGList   = "vgs"
 
-	LVCreate = "lvcreate"
-	LVRemove = "lvremove"
-	LVExtend = "lvextend"
-	LVList   = "lvs"
+	LVCreate   = "lvcreate"
+	LVRemove   = "lvremove"
+	LVExtend   = "lvextend"
+	LVList     = "lvs"
+	LVThinPool = "thin-pool"
```

Comment:
Add this `thin-pool` constant separately like `YES`, since all other constants in this section are lvm commands